### ISSUE TITLE
Add a alt="" to the hidden image

### ISSUE THREAD

**Issue by placoderm** [billerickson/Shared-Counts-Pinterest-Image#4]
https://github.com/billerickson/Shared-Counts-Pinterest-Image/blob/40b6e95a7939fdd85a1706c88bb50829830b675a/shared-counts-pinterest-image.php#L271

This is a very small thing, but I noticed that the hidden image doesn't have an alt="" attribute so that screen readers will ignore it.

[I'm new at creating issues, so I hope this was done correctly. Also, thanks for this plugin!]

**Comment by billerickson** [comment]
Thank you for reporting this issue. I actually made this fix back in November ([see here](https://github.com/billerickson/Shared-Counts-Pinterest-Image/commit/40b6e95a7939fdd85a1706c88bb50829830b675a)) but forgot to push the update to WordPress.org. The new version should be out shortly.

Thanks

**Comment by placoderm** [comment]
I was so happy to see that you were the dev on this. I couldn't make any of my sites without your display posts plugin. Thanks for being so generous with your time.
